### PR TITLE
Fix help URL

### PR DESCRIPTION
**Pull request by Parazoid**
Because hard-coding is a good idea.

### CODE REVIEW

File: server/commands.py [hunk 1]
```diff
@@ -174,7 +174,7 @@ def ooc_cmd_pos(client, arg):
 def ooc_cmd_help(client, arg):
     if len(arg) != 0:
         raise ArgumentError('This command has no arguments.')
-    help_url = 'https://github.com/AttorneyOnlineVidya/tsuserver3'
+    help_url = 'https://github.com/Fronku/Attorney-Online-Vidya/blob/master/README.md'
```
Comment (collinxchu) [inline server/commands.py]: Don't link directly to the README, follow the old format.

Comment (Parazoid) [reply server/commands.py]: Alright.

Comment (Parazoid) [reply server/commands.py]: May I ask why, though?

Comment (collinxchu) [reply server/commands.py]: Consistency and less confusion when looking for other portions of the github. Not everyone is github literate.

**Review by collinxchu — APPROVED** [review]

File: server/commands.py (hunk 1 above)
Comment (Parazoid) [reply server/commands.py]: Done.

**Pull request merged**